<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WholesaleClubMembers</class>
 <widget class="QMainWindow" name="WholesaleClubMembers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>850</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>WholesaleClubMembers</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox__MembersEntry">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>511</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string>Members Entry</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTextBrowser" name="textBrowser_MembersEntry">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>431</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_Options">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>251</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Members Options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QRadioButton" name="radioButton_Options_1">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>42</y>
        <width>173</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>1. Show Total Purchases</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Options_2">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>70</y>
        <width>170</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>2. Search Item by Name</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Options_4">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>126</y>
        <width>119</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>4. Add Member</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Options_3">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>98</y>
        <width>170</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>3. Show Expiration Date</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Options_6">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>182</y>
        <width>148</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>6. Create Purchases</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Options_5">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>154</y>
        <width>134</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>5. Delete Member</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Options_7">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>210</y>
        <width>125</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>7. Show Reports</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Options_8">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>238</y>
        <width>66</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>8. Quit</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Reports">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>430</y>
       <width>171</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Members Reports</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QRadioButton" name="radioButton_Reports_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>118</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>4. Rebate Sorted</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Reports_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>34</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>1. Print Any Day</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Reports_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>62</y>
        <width>170</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>2. Print Total</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Reports_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>146</y>
        <width>134</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>5. Amount due</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Reports_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>170</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>3. Quantity Sorted</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_EntryInfo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>340</y>
       <width>341</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Enter here:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="lineEdit_EntryInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ClearInfo">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_EnterInfo">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Name_ID">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>310</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Select here:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QRadioButton" name="radioButton_Name">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_ID">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>ID #</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Yes_No">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>310</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Select here:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="pushButton_Yes">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_No">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_EntryInfo_Option_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Enter here:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="lineEdit_EntryInfo_Option_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ClearInfo_Option_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_EnterInfo_Option_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_EntryInfo_Option_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Enter here:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="lineEdit_EntryInfo_Option_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ClearInfo_Option_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_EnterInfo_Option_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_EntryInfo_Option_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Enter here:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="lineEdit_EntryInfo_Option_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ClearInfo_Option_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_EnterInfo_Option_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_EntryInfo_Option_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Enter here:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="lineEdit_EntryInfo_Option_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ClearInfo_Option_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_EnterInfo_Option_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_EntryInfo_Option_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Enter here:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="lineEdit_EntryInfo_Option_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ClearInfo_Option_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_EnterInfo_Option_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox__DisplayReports">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>671</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string>Display Reports</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label_WarehouseLogo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>45</y>
       <width>591</width>
       <height>671</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_DisplayReports">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>581</width>
       <height>651</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1306</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWholesale_Club_Members">
    <property name="title">
     <string>Wholesale Club Members</string>
    </property>
   </widget>
   <addaction name="menuWholesale_Club_Members"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
